### PR TITLE
[WIP] Ore generation

### PR DESCRIPTION
**Pull request by Nickforall**
This is my first contribution to the glowstone project, so I would appreciate some feedback.

Todo:
- [x] Make ores spawn on the right y-coords
- [ ] Make ores spawn in veins
- [ ] Add xp drops
- [ ] Let ores spawn rare on certain levels, and common on certain levels

References
http://minecraft.gamepedia.com/Ore

### CODE REVIEW

File: src/main/java/net/glowstone/generator/populators/OrePopulator.java [hunk 1]
```diff
@@ -32,9 +32,51 @@ public void populate(World world, Random random, Chunk source) {
 
     private static void makeOres(Chunk source, Random random, int originX, int originY, int originZ, int amount, Material type) {
         for (int i = 0; i < amount; i++) {
-            int x = originX + random.nextInt(amount / 2) - amount / 4;
+        	int x = originX + random.nextInt(amount / 2) - amount / 4;
```
Comment (stuntguy3000) [inline src/main/java/net/glowstone/generator/populators/OrePopulator.java:35]: Kill it with fire! [Tab character..]

**Comment by stuntguy3000** [comment]
I recommend following coding style guidelines :)

**Comment by Nickforall** [comment]
I think that is a typo :/

**Comment by daboross** [comment]
There are quite a few tab characters (in the if statements as well), so I would make sure to replace those with spaces by completion.

Looks good besides that though!

**Comment by Nickforall** [comment]
Something must be wrong with my settings in eclipse, I set tabs to 4 spaces a few weeks ago.

Gonna fix that when i'm home.

**Comment by gdude2002** [comment]
This has Checkstyle issues: http://bamboo.gserv.me/browse/GLOW-SRV-PR286-47

**Comment by turt2live** [comment]
@Nickforall This is looking pretty inactive code-wise. If this is complete, please remove "WIP" from the title to have it reviewed.

If this is still a work in progress, please comment back stating so. 

Thanks!

(PS: Please limit your scope. The 4 tasks in your PR description are easily 4 different PRs and not as one giant PR. We're not looking to review monsters.)

**Comment by turt2live** [comment]
@Nickforall Please respond to my last message. Thanks!

**Comment by Nickforall** [comment]
You can close this, because I'm not intrested in glowstone anymore.

**Comment by turt2live** [comment]
OK :(

**Pull request closed without merging**